src/yachtzee/diet.ts

```diff
@@ -636,7 +647,7 @@ export function yachtzeeChainDiet(simOnly?: boolean): boolean {
       "orange"
     );
   }
-  if (jellyValuePerSpleen < extroValuePerSpleen && !simOnly) {
+  if (jellyValuePerSpleen < extroValuePerSpleen && !simOnly && jellyYachtzeeTurns > 0) {
```

Comment:
What's to stop this from deciding to pad out your parka-chain with jelly, then noticing that it's using jelly and aborting?

Reply:
Ah ok I think I see your concern.

I assume what you're saying is that suppose
1) we have enough organ space for jellies, but
2) jellies just aren't profitable to use
then this code here returns without running a pure parka-chain; we should instead reevaluate by setting `jellyYachtzeeTurns=0` so we can continue trudging on without returning since we can still parka-chain
(The code in its current state only addresses allowing parka-chains if we don't have enough organ space to run a jelly-chain)

I can try to fix that later